src/ui/screens/home/ViewServers.qml

```diff
@@ -46,6 +46,16 @@ Item {
         }
     }
 
+    Connections {
+        function onGoBack(item) {
+            if (getStack() === item && root.visible) {
+                menu._menuOnBackClicked();
+            }
+        }
+
+        target: VPNNavigator
+    }
```

Comment:
Looks good. What are the benefits to checking those two conditions?

Reply:
Well, getStack() returns the stackView, shared between all the ScreenHome views. But we want to have that view visible.